x-pack/filebeat/module/cisco/shared/ingest/asa-ftd-pipeline.yml

```diff
@@ -400,7 +400,14 @@ processors:
      if: "ctx._temp_.cisco.message_id == '338301'"
      field: "server.port"
      value: "{{source.port}}"
-
+ - dissect:
+     if: "ctx._temp_.cisco.message_id == '734001'"
+     field: "message"
+     pattern: "DAP: User %{user.email}, Addr %{source.ip}, Connection %{cisco.connection_type}: The following DAP records were selected for this connection: %{cisco.dap_records->}"
+ - split:
+     field: "cisco.dap_records"
```

Comment:
Same here. The field to split is `_temp_.cisco.dap_records`.

Also this processor needs a `ignore_missing: true` because it runs for all cisco events, and only `734001` has the field. It can also do with a condition but it's better to use ignore_missing in this case.